src/services/sessionService.ts

```diff
@@ -75,6 +79,28 @@ class Session implements SessionInterface {
       throw error;
     }
   }
+
+  private validateIssuer(jwtIssuer: string) {
+    if (!jwtIssuer) {
```

Comment:
does this include null, undefined and ""?

Reply:
Yes, all 3 cases

Reply:
okay nice :)